### ISSUE TITLE
Fails on Cordova because it uses file protocol

### ISSUE THREAD

**Issue by v3ss0n** [pitchtarget/ember-cli-facebook-js-sdk#15]
Failed to load resource: net::ERR_FILE_NOT_FOUND

**Comment by bugant** [comment]
Can you be more precise? Do you have a stack trace?

**Comment by v3ss0n** [comment]
The problem is here : 
```
      Ember.$.getScript('//connect.facebook.net/en_US/sdk.js', function() {
```
it uses the `file://connect.facebook.net` instead of `https`

**Comment by bugant** [comment]
OK will fix it next week and release a new version! Thank you for the bug report!

**Comment by bugant** [comment]
@v3ss0n it should be fixed on v1.0.5 which is being published now. Please, re-open the issue if you think the fix is not correct or you still have problem with it.
